services/management/ia/templates_service.go

```diff
@@ -42,70 +43,44 @@ import (
 	"gopkg.in/reform.v1"
 	"gopkg.in/yaml.v3"
 
-	"github.com/percona/pmm-managed/data"
 	"github.com/percona/pmm-managed/models"
-	"github.com/percona/pmm-managed/utils/dir"
 )
 
 const (
-	templatesParentDir = "/srv/ia"
-	templatesDir       = "/srv/ia/templates"
-	rulesParentDir     = "/etc/ia"
-	rulesDir           = "/etc/ia/rules"
-	dirPerm            = os.FileMode(0o775)
+	builtinTemplatesPath = "/tmp/ia1/*.yml"
```

Comment:
is it safe to use `/tmp/` directory?

Reply:
That's a result of incorrect merging and conflicts resolving too